src/calculateRentalCost.js

```diff
@@ -29,6 +29,16 @@
 
 function calculateRentalCost(days) {
   // write code here
+
+  if (days >= 3 && days < 7) {
```

Comment:
swap your if's blocks order so you can omit second check
```suggestion
  if (days >= 3) {
```

Reply:
> swap your if's blocks order so you can omit second check

Thank you for your advice!)